com.playeveryware.eos/Documentation~/google_signin.md

```diff
@@ -0,0 +1,54 @@
+<a href="/com.playeveryware.eos/README.md"><img src="/com.playeveryware.eos/Documentation~/images/PlayEveryWareLogo.gif" alt="README.md" width="5%"/></a>
+
+# <div align="center">Connect Login with GoogleID using Credential Manager</div>
+---
+
+## Overview
+
+This document describes how to enable Google ID authentication in conjunction with Epic Online Services Plugin for Unity.  
+We followed [Authenticate users with Sign in with Google](https://developer.android.com/identity/sign-in/credential-manager-siwg) and implemented a plugin for the users to easily achieve the feature.
+
+## Setup
+
+### Plugins
+The feature is included in our samples, so no external plugin is needed.
+
+### Build Settings
+To allow the feature, follow the build configuration step below. 
+Add the following in the dependencies of the project's main gradle file. (The default is `mainTemplate.gradle`) 
+```
+        implementation 'androidx.credentials:credentials-play-services-auth:1.3.0'
+        implementation 'androidx.credentials:credentials:1.3.0'
+        implementation 'com.google.android.libraries.identity.googleid:googleid:1.1.1'
+```
+
+### Configurations
+Configurate `GoogleLoginClientID` and optionally`GoogleLoginNonce` in `AndroidConfig`.   
```

Comment:
I do not know if "Configurate" is a word, but it's a cool thing to say. Might need to be "Configure"

Reply:
Fixed !